### DISCUSSION TITLE
WebSocket SentFrame masking only first frame

### DISCUSSION THREAD

**Discussion by bezdziel** [nanoframework/Home#1513] [General]
Hi, I have a problem with receiving raw image data sending via WebSockets. I've investigated that only first frame is correctly masked. Rest of frames are skipped cause of invalid condition. Below you can see a screenshoot from WebSocketSender.cs (see watch values)
![image](https://github.com/nanoframework/Home/assets/24956773/eb466984-4fea-4f36-9841-137064ec17da)

**Comment by josesimoes** [comment]
pinging @Feiko

**Comment by bezdziel** [comment]
Any news here?

**Reply by josesimoes** [reply]
Nothing. Please hold as our WebSockets expert is on holiday. Returning September.
